Configure.pl

```diff
@@ -430,6 +430,9 @@ sub uniq {
 push @cflags, '-DMVM_HEAPSNAPSHOT_FORMAT=' . $config{heapsnapformat};
 push @cflags, $ENV{CFLAGS} if $ENV{CFLAGS};
 push @cflags, $ENV{CPPFLAGS} if $ENV{CPPFLAGS};
+
+push @cflags, '-D_GNU_SOURCE'; # mainly for libuv
```

Comment:
What about making this `push @cflags, '-D_GNU_SOURCE' unless $args{'has-libuv'};`?

Reply:
Okay, will do.